test/e2e/README.md

```diff
@@ -26,6 +26,10 @@ The `TEST_E2E_PODVM_IMAGE` is an optional variable which specifies the path to t
 $ TEST_E2E_PROVISION=yes TEST_E2E_PODVM_IMAGE="path/to/podvm-base.qcow2" CLOUD_PROVIDER=libvirt make test-e2e
 ```
 
+By default it is given 20 minutes for the entire e2e execution to complete, otherwise the process is preempted. If you need to extend that timeout then export the `TEST_E2E_TIMEOUT` variable. For example, `TEST_E2E_TIMEOUT=30m` set the timeout to 30 minutes. See `-timeout` in [go test flags](https://pkg.go.dev/cmd/go#hdr-Testing_flags) for the values accepted.
+
+To leave the cluster untouched by the execution finish you should export `TEST_E2E_TEARDOWN=no`, otherwise the framework will attempt to wipe out any resources created. For example, if `TEST_E2E_PROVISION=yes` then it will delete both the cluster and VPC.
+
```

Comment:
nit: TEST_E2E_PROVISION=yes => TEST_E2E_TEARDOWN=yes

Reply:
So I think Wainer's doc is actually correct, but maybe could do with some more clarification e.g.
> For example, if `TEST_E2E_PROVISION=yes` it used to create a VPC and cluster for testing and `TEST_E2E_TEARDOWN=no` not specified, then at the end of the test the the provisioned cluster and VPC, will both be deleted.

I'm not sure if that helps @bpradipt and if I've interpreted it correctly @wainersm?

Reply:
Thanks @stevenhorsman, the example you gave makes sense.

Reply:
Replaced the example with the @stevenhorsman 's.